nmmo/core/env.py

```diff
@@ -398,11 +411,15 @@ def _compute_rewards(self, agents: List[AgentID], dones: Dict[AgentID, bool]):
           entity identified by ent_id.
     '''
     # Initialization
-    infos = {agent_id: {'task': {}} for agent_id in agents}
-    rewards = defaultdict(int)
-    agents = set(agents)
+    infos = {}
+    rewards = {}
     reward_cache = {}
 
+    for eid in self.possible_agents:
+      infos[eid] = {}
+      infos[eid]['task'] = {}
+      rewards[eid] = 0
```

Comment:
this will add rewards for every agent, alive or dead.  ... and looks like old code?

I removed the code for removing rewards for the dead agent